src/theme/breakpoints.js

```diff
@@ -1,4 +1,6 @@
 const breakpoints = {
+  'mobile': '320px',
+  'tablet': '740px',
```

Comment:
As we only have mobile and desktop viewports, can you remove this? 🙏

Reply:
Okkay will remove that tablet part !

Reply:
Thanks! You can also remove the `mobile` part, as mobile will be the default format (mobile first!) 🥳

Reply:
Sorry if I wasn't clear enough 😄 

What I meant was:
```js
const Logo = styled.img`
  display: flex;
  width: 180px;
  @media (min-width: ${get('breakpoints.desktop')}) {
    width: 250px;
  }
`
```

This means `by default the image is 180px wide, but once it's on desktop viewport, it's 250px wide`. Hope it helps! 😉

Reply:
Ohhhhh 😯 i will make changes and push it. Thanks for clarification ☺️